crmsh/sbd.py

```diff
@@ -150,8 +154,13 @@ def _initialize_sbd(self):
         """
         if self.diskless_sbd:
             return
+        opt = ""
+        if self._context.profiles_dict:
+            sbd_msgwait = self._context.profiles_dict["sbd.msgwait"]
+            self._sbd_watchdog_timeout = self._context.profiles_dict["sbd.watchdog_timeout"]
+            opt = "-4 {} -1 {}".format(sbd_msgwait, self._sbd_watchdog_timeout)
```

Comment:
what if no msgwait is explicitly set in the profiles?

Reply:
Changed the logic, and removed the sbd.msgwait from profiles.yml
only detected if "sbd.watchdog_timeout" in profiles.yml
then msgwait will be calculated

Reply:
I like the idea of calculating msgwait. But it doesn't seem to correctly handle the case where  `sbd.msgwait`  is set in profile but `sbd.watchdog_timeout` doesn't exist ...

I guess this could be a good place to reference predefined `SBD_WATCHDOG_TIMEOUT_DEFAULT` and `SBD_WATCHDOG_TIMEOUT_DEFAULT_s390`